docs/configuration.md

```diff
@@ -2018,6 +2018,50 @@ Apart from these, the following properties are also available, and may be useful
 </tr>
 </table>
 
+### Barrier Execution Mode
+
+<table class="table">
+<tr><th>Property Name></th><th>Default</th><th>Meaning</th></tr>
```

Comment:
Shall we remove the extra `>` at `Property Name>`?
If you don't mind, please generate the doc and post the screenshot into the PR description.
That is helpful to spot this kind of typo-related bugs and the broken tables.

Reply:
Thanks. I've attached below.